worlds/_bizhawk/client.py

```diff
@@ -5,7 +5,7 @@
 from __future__ import annotations
 
 import abc
-from typing import TYPE_CHECKING, Any, ClassVar, Dict, Optional, Tuple, Union
+from typing import TYPE_CHECKING, Any, ClassVar, Optional, Union
```

Comment:
```suggestion
from typing import TYPE_CHECKING, Any, ClassVar
```
Similarly, union can be expressed as `<Type> | <Type>`. Unsure if all of the lines that need changing are in this file, so you would have to find other usages yourself.